EmployeeManagement/Sources/openai_request.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+// Hardcoded OpenAI API Key (not recommended)
+let apiKey = "sk-utRJzHluRSPOljnDhnORT5BlbkFJkPSHrIf6MqUIs1rAfUY0"
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider securing the API key</b></div>
<div id="fix">

Consider moving the API key to a secure configuration file or environment variable instead of hardcoding it in the source code. Exposing API keys in source code poses a security risk. A similar issue was also found in EmployeeManagement/Sources/employee.swift (line 5-10).
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -3,6 +3,6 @@

  // Hardcoded OpenAI API Key (not recommended)
 -let apiKey = "sk-utRJzHluRSPOljnDhnORT5BlbkFJkPSHrIf6MqUIs1rAfUY0"
 +let apiKey = ProcessInfo.processInfo.environment["OPENAI_API_KEY"] ?? ""
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/Bito-CRA-Issues-Dataset/pull/10#issuecomment-2514547000>#0b17a0</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged